djstripe/models.py

```diff
@@ -50,7 +51,9 @@
 
 @class_doc_inherit
 class Charge(StripeCharge):
-    __doc__ = getattr(StripeCharge, "__doc__")
+    """A model extending StripeCharge providing dj-stripe functionality."""
```

Comment:
https://github.com/PyCQA/pydocstyle/issues/97 :/

Maybe a better docstring to alleviate this?